draft/DRAFT_TEMPLATE

```diff
@@ -104,13 +104,18 @@ Are you a new or experienced speaker looking for a place to share something cool
 
 If you are an event organizer hoping to expand the reach of your event, please submit a link to the website through a [PR to TWiR](https://github.com/rust-lang/this-week-in-rust) or by reaching out on [X (formerly Twitter)](https://x.com/ThisWeekInRust) or [Mastodon](https://mastodon.social/@thisweekinrust)!
 
+### Rust Compiler Performance Triage
```

Comment:
Moving this up without any other changes makes it a child of the Call for Participation heading.

Reply:
I missed the difference between `##` and `###`, it was confusing to me that "Updates from the Rust Project" just included a list of PRs :man_shrugging: Thanks!